lint/lint.py

```diff
@@ -63,6 +63,18 @@ def check_path_length(repo_root, path):
     return []
 
 
+def check_worker_collision(repo_root, path):
+    endings = [(".any.html", ".any.js"),
+               (".any.worker.html", ".any.js"),
+               (".worker.html", ".worker.js")]
+    for path_ending, generated in endings:
+        if path.endswith(path_ending):
+            return [("WORKER COLLISION", "path ends with %s which collides with generated tests from %s files" % (path_ending, generated),
```

Comment:
One more nit: please put each of the tuple fields on its own line, rather than having the first two together here.